README.md

```diff
@@ -21,6 +21,13 @@ For example, if you see `types "sigs.k8s.io/cluster-api/vendor/k8s.io/apimachine
 
 You can use Google Cloud or AWS ECR to host repositories.
 
+1. Install [jq](https://stedolan.github.io/jq/download/) (`brew install jq` on Mac)
+2. Install `gettext` package (`brew install gettext && brew link --force gettext` on Mac)
+3. Install `minikube` (`brew install minikube`)
```

Comment:
think it makes sense to link out to minikube and gettext for download/installation instructions for other platforms? I like that we have specific instructions for Mac though